r/tests/testthat/helper-arrow.R

```diff
@@ -18,12 +18,6 @@
 # Wrap testthat::test_that with a check for the C++ library
 options(..skip.tests = !arrow:::arrow_available())
 
-if (tolower(Sys.info()[["sysname"]]) == "windows") {
-  # For now, disable multithreading by default on Windows
-  # See https://issues.apache.org/jira/browse/ARROW-8379
-  options(arrow.use_threads = FALSE)
-}
```

Comment:
I've copy/pasted this chunk into ticket https://issues.apache.org/jira/browse/ARROW-14159?focusedCommentId=17422217&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17422217 so it's obvious (one way to) make this happen when we get there